allennlp/training/metrics/span_based_f1_measure.py

```diff
@@ -69,19 +70,30 @@ def __call__(self,
             shape as the ``predictions`` tensor without the ``num_classes`` dimension.
         mask: ``torch.Tensor``, optional (default = None).
             A masking tensor the same size as ``gold_labels``.
+        prediction_map: ``torch.Tensor``, optional (default = None).
```

Comment:
Could you add a little comment about the use case for this, i.e that you need it for models which produce distributions over tags conditioned on the frame, just because it's a little hard to imagine why you would need this without a little context.